_pages/cfp.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
this page should be linked to from the speakers page and home page where it's already mentioned (I think?)